core-infrastructure/src/db/Core.Database/Scripts/007-CreateTrustTable.sql

```diff
@@ -0,0 +1,19 @@
+IF NOT EXISTS(SELECT *
+              FROM INFORMATION_SCHEMA.TABLES
+              WHERE table_name = 'Trust')
+    BEGIN
+        CREATE TABLE dbo.Trust
+        (
+            UKPRN         nvarchar(50)  NOT NULL,
```

Comment:
Should we distinguish the Trust UKPRN from the UKPRN?

Reply:
It's the trust table, so we don't need it.